new-log-viewer/src/utils/actions.ts

```diff
@@ -0,0 +1,65 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
+
+import {Nullable} from "../typings/common";
+
+
+enum ACTION {
+    FIRST_PAGE = "firstPage",
+    PREV_PAGE = "prevPage",
+    NEXT_PAGE = "nextPage",
+    LAST_PAGE = "lastPage",
+    PAGE_TOP = "pageTop",
+    PAGE_BOTTOM = "pageBottom"
+}
+
+type ActionType = {
+    action: Nullable<ACTION>,
+    label: string,
+    keybindings: monaco.KeyCode[],
+}
+
+/* eslint-disable sort-keys */
+const EDITOR_ACTIONS : ActionType[] = [
+    {
+        action: null,
```

Comment:
I haven't read through all the code, but so far I'm not sure why we allow action to be `null` (in the same vein as why we can't handle Backquote with `setupCustomActions`).

Reply:
> why we can't handle Backquote with `setupCustomActions`

When the monaco container is not focused, any keydown events are not picked up by Monaco and therefore no Monaco customer actions can be triggered.

Reply:
Right. So then do we need the action for the backquote?

Reply:
Discussed offline - to be used in rendering a help menu for shortcuts. It might be better to document why we have null values here though.